code/mapentities/TTTCreditAdjust.cs

```diff
@@ -0,0 +1,43 @@
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.MapEntities
+{
+    [Library("ttt_credit_adjust", Description = "Changes the amount of credits upon the activator.")]
+    public partial class TTTChangeCredits : Entity
+    {
+        [Property("Credits", "Removes amount of credits. Use a negative number to add credits to the activator.")]
+        public int Credits { get; set; } = 0;
```

Comment:
I wouldn't design it the way you just can remove credits, instead, a general approach would be cool (possibility to add credits as well).

Reply:
I prefer the logic this way as it is how it worked in the original TTT. I've gone ahead and changed the property description to be a bit more specific.